pkg/pb/contacts.proto

```diff
@@ -8,15 +8,26 @@ import "github.com/lyft/protoc-gen-validate/validate/validate.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 import "github.com/infobloxopen/protoc-gen-gorm/options/gorm.proto";
 
-option go_package = "github.com/infobloxopen/atlas-contacts-app/pb/contacts;contacts";
+option go_package = "github.com/infobloxopen/atlas-contacts-app/pkg/pb;pb";
 
 message Contact {
     option (gorm.opts).ormable = true;
     uint64 id = 1;
     string first_name = 2;
     string middle_name = 3;
     string last_name = 4;
-    string email_address = 5 [(validate.rules).string.email = true];
+    string primary_email = 6 [(gorm.field).drop = true, (validate.rules).string.email = true];
```

Comment:
great job skipping the 5 -- you're a true protobuffer!

Reply:
:+1: